js/src/secureApi.js

```diff
@@ -25,12 +25,13 @@ export default class SecureApi extends Api {
     this._isConnecting = true;
     this._connectState = sysuiToken === 'initial' ? 1 : 0;
     this._needsToken = false;
-    this._nextToken = nextToken;
     this._dappsPort = 8080;
     this._dappsInterface = null;
     this._signerPort = 8180;
+    this._followConnectionTimeoutId = null;
 
-    console.log('SecureApi:constructor', sysuiToken);
+    // Try tokens from localstorage, then from hash
+    this._tokensToTry = [ sysuiToken, nextToken ].filter((t) => t && t.length);
```

Comment:
Ok, one last comment - as soon as we change anything here we need a full test of all the iterations -

1. token cleared, none specified
2. token set to initial in localStorage`
3. localeStorage token, none specified
4. invalid localStorage token
5. 1, 2, 3 & 4 with `parity ui` (calling /#/auth)
6. doing everything wrong and having to supply a token (both correct & incorrect)
7. ...

e.g. also see we have autoConnect swapped in WS, could be fine with logic, but need to thoroughly re-test all scenarios we can think of here.